ClinGen/clincoded
# Connects to #1330. Adding evidence summary text box in VCI.

**Steps to test:**
1. Start a new VCI interpretation and evaluate some of the criteria so that a calculated pathogenicity is generated.
2. Click the **View Summary** button at the upper right hand corner to load the VCI Evaluation Summary page.
3. On the summary page, expect to see a new "Evidence Summary" label and its text field.
4. Enter some text into the "Evidence Summary" text field and click the **Save** button. Expect to see the form to be successfully saved and a green alert appearing next the button upon successful form submission (and disappearing in 6 seconds).
![image](https://user-images.githubusercontent.com/6956310/29334192-6f28b12c-81bb-11e7-8b16-13ab0a0c52bf.png)
5. Click on the blue **?** icon with the "Mark status..." label above the text field and expect to see a popover appearing with new **Help** description. Click on the **?** icon again and expect to see it disappearing.
![image](https://user-images.githubusercontent.com/6956310/29334277-ba39cd7c-81bb-11e7-9969-8dafb11d50ee.png)
6. Expect to see that the yellow warning text box about "disease needed to be associated for Likely Pathogenicity or Pathogenicity" is no longer present below the blue **?** icon with the "Mark status..." label.
